identity-service/src/app.js

```diff
@@ -294,11 +295,48 @@ class App {
       ethRelayWalletRateLimiter,
       ethRelayIPRateLimiter
     )
+
+    // POA relay rate limits
+    const poaRelayWalletRateLimiter = getRateLimiter({
+      prefix: `poaRelayWalletRateLimiter`,
+      expiry: ONE_HOUR_IN_SECONDS,
+      max: config.get('rateLimitingPOARelaysPerWalletPerHour'),
+      statusCode: 211, // don't return a 429, so libs won't retry
```

Comment:
What's the meaning of 211 here instead of just 200?

Reply:
agree - can we update libs to respect the 429 in the case of the relay versus a generic 4xx/5xx